specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2021-03-01-preview/mfe.json

```diff
@@ -0,0 +1,4932 @@
+            "name": "version",
+            "description": "Version identifier.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/WorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ModelVersionResource"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Model Version.": {
+            "$ref": "./examples/ModelVersion/get.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ModelVersion"
+        ],
+        "summary": "Delete version.",
+        "operationId": "ModelVersions_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "name",
+            "description": "Container name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "version",
+            "description": "Version identifier.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/WorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          },
+          "200": {
+            "description": "Success"
+          },
+          "204": {
+            "description": "Success"
+          }
+        },
+        "x-ms-examples": {
+          "Delete Model Version.": {
+            "$ref": "./examples/ModelVersion/delete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CodeContainer": {
+      "type": "object",
+      "properties": {
+        "properties": {
```

Comment:
the schema of CodeContainers can not be more tightly defined than this?

Reply:
my mistake, there's some confusion here. The x-ms-client-flatten should be one level higher. The inner "properties" is meant to be the name of the field that is alongside "tags" and "description".

With that in mind, I would think that's as tight as it can be

Reply:
wait sorry no, ignore my previous comment. This is the proper intention. But no, I don't think this can be more tightly defined